lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
+  <h2>${_("Entrance Exam grade adjustment")}</h2>
+  <div class="request-response-error"></div>
+  <p>
+  ${_("Specify the {platform_name} email address or username of a student here:").format(platform_name=settings.PLATFORM_NAME)}
+  <input type="text" name="entrance-exam-student-select-grade" placeholder="${_("Student Email or Username")}">
+  </p>
+  <br>
```

Comment:
Thanks @frrrances. What happened here is that this is duplicated HTML from another part of the instructor dashboard that was already this bad. @marcotuts tells me that he's happy to accept the tech debt rather than having to clean up the instructor dashboard at this point, and @explorerleslie has signed off on this approach.